app/UseCases/Curator/ListCuratorUseCase.js

```diff
@@ -0,0 +1,23 @@
+const NotFoundException = use('App/Exceptions/NotFoundException');
+
+class GetCuratorUseCase {
+  static get inject() {
+    return ['App/Models/Curator'];
+  }
+
+  constructor(curatorModel) {
+    this.curatorModel = curatorModel;
+  }
+
+  async execute() {
+    const curator = await this.curatorModel.all();
+
+    if (!curator) {
```

Comment:
Retirar essa validação pois se nenhum curador for encontrado é retornado um array vazio